packages/core/src/App/Containers/AccountSwitcherWallet/account-switcher-wallet.tsx

```diff
@@ -21,8 +21,16 @@ export const AccountSwitcherWallet = observer(({ is_visible, toggle }: TAccountS
 
     const wrapper_ref = React.useRef<HTMLDivElement>(null);
 
-    const validateClickOutside = (event: MouseEvent) =>
-        is_visible && !(event.target as unknown as HTMLElement).classList.contains('acc-info');
+    const validateClickOutside = (event: MouseEvent) => {
+        const checkAllParentNodes = (node: HTMLElement): boolean => {
+            if (node?.classList?.contains('acc-info__wallets')) return true;
+            const parent = node?.parentNode as HTMLElement;
+            if (parent) return checkAllParentNodes(parent);
+            return false;
+        };
+
+        return is_visible && !checkAllParentNodes(event.target as unknown as HTMLElement);
```

Comment:
minor: "event.target as unknown as HTMLElement" looks bit weird, but I guess its fine if not viable alternative is out there,

Reply:
@wojciech-deriv I didn't know how to say TS that it's a node, just this ugly way... If you have some idea I'd appreciate it

Reply:
(it dose seem to work locally on mine machine)